### PR TITLE
chore(scripts): add script to install awscli on Debian

### PR DESCRIPTION
**Pull request by mloberg**
Adds a script to install awscli@v2 while keeping the smallest layer size. Any tools it needs to install are then removed, but only if this script installed them

### CODE REVIEW

File: scripts/awscli.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+AWSCLI_VERSION="${AWSCLI_VERSION:-2.9.15}"
+
+missing=""
+command -v curl >/dev/null || missing="${missing} curl"
+command -v unzip >/dev/null || missing="${missing} unzip"
+test -d /etc/ssl/certs || missing="${missing} ca-certificates"
+
+if [ "$missing" != "" ]; then
+    apt-get update -qq
```
Comment (dholmes) [inline scripts/awscli.sh:12]: Q: do we have any non-debian/apt images to worry about?

**Review by dholmes — APPROVED** [review]
Assuming we only need apt-get for now, but asked the question just in case.

File: scripts/awscli.sh (hunk 1 above)
Comment (mloberg) [reply scripts/awscli.sh:12]: Alpine images don't support awscli@v2, but have an `aws-cli` package for the few that we have. We don't have enough Red Hat images at the moment to worry about adding support for it here

Comment (dholmes) [reply scripts/awscli.sh:12]: Perfect! Thank you for the info.

**Pull request merged**